src/state/experiments/selectors.js

```diff
@@ -1,2 +1,41 @@
 export const getExperimentsCount = state =>
   state.get("experiments").get("items").size;
+
+export const getFilteredExperimentsByDate = (state) => {
+  const experiments = state.get("experiments").get("items"); 
```

Comment:
This looks like it could be another selector?

Reply:
Any time you are getting nested values you should use `.getIn` instead of multiple `.get`